src/pages/Search/SearchFiltersCategoryPage.tsx

```diff
@@ -0,0 +1,96 @@
+import React, {useMemo} from 'react';
+import {View} from 'react-native';
+import {useOnyx} from 'react-native-onyx';
+import FullPageNotFoundView from '@components/BlockingViews/FullPageNotFoundView';
+import type {FormOnyxValues} from '@components/Form/types';
+import HeaderWithBackButton from '@components/HeaderWithBackButton';
+import ScreenWrapper from '@components/ScreenWrapper';
+import SelectionList from '@components/SelectionList';
+import RadioListItem from '@components/SelectionList/RadioListItem';
+import useLocalize from '@hooks/useLocalize';
+import useThemeStyles from '@hooks/useThemeStyles';
+import localeCompare from '@libs/LocaleCompare';
+import * as SearchActions from '@userActions/Search';
+import ONYXKEYS from '@src/ONYXKEYS';
+
+function SearchFiltersStatusPage() {
+    const styles = useThemeStyles();
+    const {translate} = useLocalize();
+
+    const [searchAdvancedFiltersForm] = useOnyx(ONYXKEYS.FORMS.SEARCH_ADVANCED_FILTERS_FORM);
+    const activeItems = searchAdvancedFiltersForm?.category;
+    const policyId = searchAdvancedFiltersForm?.policyId ?? '-1';
+
+    const [callCategories] = useOnyx(ONYXKEYS.COLLECTION.POLICY_CATEGORIES);
+    const [singlePolicyCategories] = useOnyx(`${ONYXKEYS.COLLECTION.POLICY_CATEGORIES}${policyId}`);
+
+    let categories: string[] = [];
+    if (!singlePolicyCategories) {
+        categories = Object.values(callCategories ?? {})
+            .map((value) => Object.values(value ?? {}))
+            .map((value2) => value2.map((value3) => value3?.name))
+            .flat();
+    } else {
+        categories = Object.values(singlePolicyCategories ?? {}).map((value) => value.name);
+    }
+
+    const removeDuplicates = (array: string[]) => array.filter((value, index) => array.indexOf(value) === index);
```

Comment:
Helper functions like this one shouldn't be defined inside component body. You need to move it outside.

Also I think there is a simpler way to do this, we can use the fact that `Set` removes duplicates and do it like this:
```ts
const deduplicatedData = [...new Set(data)]
```

then you don't need an extra function I think